flash/image/classification/input_transform.py

```diff
@@ -50,9 +55,14 @@ def input_per_sample_transform(self):
         return T.Compose([T.ToTensor(), T.Resize(self.image_size), T.Normalize(self.mean, self.std)])
 
     def train_input_per_sample_transform(self):
-        return T.Compose(
-            [T.ToTensor(), T.Resize(self.image_size), T.Normalize(self.mean, self.std), T.RandomHorizontalFlip()]
-        )
+        transforms = [T.TrivialAugmentWide()] if _TORCHVISION_GREATER_EQUAL_0_11 else []
```

Comment:
This seems really weird to me. It means that for different PL versions you will get different augmentations, so also potentially quite dramatically different performance...